recipes/taocpp-pegtl/3.x.x/conanfile.py

```diff
@@ -14,39 +15,55 @@ class TaoCPPPEGTLConan(ConanFile):
               "parsing", "cpp17", "cpp11", "grammar")
     no_copy_source = True
     settings = "compiler"
+    options = {
+        "boost_filesystem": [True, False],
```

Comment:
Would a `with_` prefix be more consistent with other recipes pulling in external dependencies?

Reply:
yes, but it suggests that we enable a feature, which is not the case, it's more a backend.

`filesystem_backend: ["std", "experimental", "boost"]`?

Reply:
That makes sense. Thanks.